app/assets/locales/messages_ccodk_default.txt

```diff
@@ -79,7 +79,7 @@ home.forms.incomplete=Incomplete
 home.forms.incomplete.indicator=${1} (${0})  
 home.forms.saved=Saved
 home.sync=Sync with Server
-home.sync.indicator=${1} (${0})
+home.sync.indicator=Unsent Forms: ${0}

```

Comment:
Curious what happens when people don't update their app translations when updating to 2.27 and we try to localize `${1} (${0})`? Can we make sure we fail soft?